We want to use our own version info. Before JVMCI is initialized, it performs a version check.
It expects the JVMCI build revision to be appended to the JDK version. Never fail the version
check so that we don't have to worry about manually aligning version info each release to match
what the check wants from us

diff --git a/graal/compiler/src/jdk.graal.compiler/src/jdk/graal/compiler/hotspot/JVMCIVersionCheck.java b/graal/compiler/src/jdk.graal.compiler/src/jdk/graal/compiler/hotspot/JVMCIVersionCheck.java
index ea6f3b4d30e..1181df69ce0 100644
--- a/graal/compiler/src/jdk.graal.compiler/src/jdk/graal/compiler/hotspot/JVMCIVersionCheck.java
+++ b/graal/compiler/src/jdk.graal.compiler/src/jdk/graal/compiler/hotspot/JVMCIVersionCheck.java
@@ -299,17 +299,7 @@ public final class JVMCIVersionCheck {
     }
 
     private static void failVersionCheck(boolean exit, String errorMessage) {
-        String value = System.getenv("JVMCI_VERSION_CHECK");
-        if ("warn".equals(value)) {
-            System.err.println(errorMessage);
-        } else if ("ignore".equals(value)) {
-            return;
-        } else if (exit) {
-            System.err.println(errorMessage);
-            System.exit(-1);
-        } else {
-            throw new InternalError(errorMessage);
-        }
+	return;
     }
 
     /**
